orders/convoy.go

```diff
@@ -84,7 +84,7 @@ func (self *convoy) Options(v godip.Validator, nation godip.Nation, src godip.Pr
 	if !ok || convoyer.Type != godip.Fleet {
 		return
 	}
-	if v.Graph().Flags(actualSrc)[godip.Land] && !v.Graph().Flags(actualSrc)[godip.Convoyable] {
+	if v.Graph().Flags(actualSrc.Super())[godip.Land] && !v.Graph().Flags(actualSrc.Super())[godip.Convoyable] {
```

Comment:
Is this safe? Do all super-provinces have the correct Land/Convoyable flags?

Reply:
Yes, I'm pretty sure they do.

The coast subprovinces only have Sea set because fleets are allowed there and armies aren't, and vice versa for the super provinces.

Reply:
Alright then :)